src/Microsoft.Azure.WebJobs.Host/Bindings/FunctionBindingContext.cs

```diff
@@ -26,12 +29,13 @@ public class FunctionBindingContext
         public FunctionBindingContext(
             Guid functionInstanceId,
             CancellationToken functionCancellationToken,
+            IServiceProvider functionInvocationServices = null,
             FunctionDescriptor functionDescriptor = null)
         {
```

Comment:
This is a public class - isn't this a breaking change to the ctor?

Reply:
Sorry, I should have marked this as a draft, but wanted to make sure that wasn't going to impact or packaging. This will be a proper overload.